src/components/routerLink.browser.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { mount } from '@vue/test-utils'
+import { expect, test, vi } from 'vitest'
+import { h } from 'vue'
+import routerLink from '@/components/routerLink'
+import { Route } from '@/types'
+import { component, createRouter } from '@/utilities'
+
+test('renders an anchor tag with the correct href and slot content', () => {
+  const path = 'path/:param'
+  const param = 'param'
+  const content = 'hello world'
+  const href = path.replace(':param', param)
+
+  const route = {
+    name: 'parent',
+    path,
+    component,
+  } as const satisfies Route
+
+  const router = createRouter([route], {
+    initialUrl: route.path,
```

Comment:
if this spec is running in happydom do we still need to clarify an `initialURL`?

Reply:
I think long term that was the vision, maybe we just didn't implement the "go get it from the browser" part yet

Reply:
Correct. But for a unit test it's also nice to explicitly set it to the route defined rather than just magically matching the window.location